components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/RequestObjectValidatorUtil.java

```diff
@@ -245,4 +289,22 @@ public static boolean isSignatureVerified(SignedJWT signedJWT, Certificate x509C
             return false;
         }
     }
+
+    public static JWSVerifier getVerifier(PublicKey publicKey) {
+
+        return new RSASSAVerifier((RSAPublicKey) publicKey);
+
+    }
+
+    public static boolean isFapiConformant(String clientId) throws RequestObjectException {
```

Comment:
why is this method public?

Reply:
Since this is just calling the OAuth util method lets not create another util method for it. instead, use the oauth2util method in the relevant places or create private methods in the needed classes

Reply:
Fixed in f409e32a5e8cad00a5dfeaaac027011720f2c534